collectors/COLLECTORS.md

```diff
@@ -1,219 +1,478 @@
+-   [Postfix](/collectors/python.d.plugin/postfix/README.md): Uses the `postqueue` tool to monitor the queue length of a
+    mail/message transfer agent (MTA).
+
+### Kubernetes
+
+-   [Kubelet](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/k8s_kubelet/): Monitor one or more
+    instances of the Kubelet agent and collects metrics on number of pods/containers running, volume of Docker
+    operations, and more.
+-   [kube-proxy](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/k8s_kubeproxy/): Collect
+    metrics, such as syncing proxy rules and REST client requests, from one or more instances of `kube-proxy`.
+-   [Service discovery](https://github.com/netdata/agent-service-discovery/): Find what services are running on a
+    cluster's pods, converts that into configuration files, and exports them so they can be monitored by Netdata.
+
+### Logs
+
+-   [Fluentd](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/fluentd/): Gather application
+    plugins metrics from an endpoint provided by `in_monitor plugin`.
+-   [Logstash](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/logstash/): Monitor JVM threads,
+    memory usage, garbage collection statistics, and more.
+-   [OpenVPN status logs](/collectors/python.d.plugin/ovpn_status_log/): Parse server log files and provide summary
+    (client, traffic) metrics.
+-   [Web server logs (Go version for Apache,
+    NGINX)](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/weblog/): Tail access logs and provide
+    very detailed web server performance statistics. This module is able to parse 200k+ rows in less than half a second.
+-   [Web server logs (Python version for Apache, NGINX, Squid)](/collectors/python.d.plugin/web_log/): Tail access log
+    file and collect web server/caching proxy metrics.
+
+### Messaging
+
+-   [ActiveMQ](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/activemq/): Collect message broker
+    queues and topics statistics using the ActiveMQ Console API.
+-   [Beanstalk](/collectors/python.d.plugin/beanstalk/README.md): Collect server and tube-level statistics, such as CPU
+    usage, jobs rates, commands, and more.
+-   [Pulsar](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/pulsar/): Collect summary,
+    namespaces, and topics performance statistics.
+-   [RabbitMQ (Go)](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/rabbitmq/): Collect message
+    broker overview, system and per virtual host metrics.
+-   [RabbitMQ (Python)](/collectors/python.d.plugin/rabbitmq/README.md): Collect message broker global and per virtual
+    host metrics. 
+-   [VerneMQ](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/vernemq/): Monitor MQTT broker
+    health and performance metrics. It collects all available info for both MQTTv3 and v5 communication
+
+### Network
```

Comment:
What does `network` means? i see it contains:

- dhcp
- dns
- ntp
- vpn (openvpn, libreswan)
- aaa (freeradius)
- voice/video (openSIPS)

Reply:
It's kind of a catchall category for anything to do with networks and/or sending data across networks, and doesn't fit better elsewhere. I'm trying to minimize the number of categories/types as much as possible.

Reply:
> a catchall category
> sending data across networks
> trying to minimize the number of categories/types as much as possible.

Yeah, i agree, it is either you have a lot of categories or have _a catchall category_.